packages/ast-spec/tests/fixtures-with-differences-errors.shot

```diff
@@ -0,0 +1,16 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AST Fixtures List fixtures with Error differences 1`] = `
+Object {
+  "Babel errored but TSESTree didn't": Set {
+    "declaration/ClassDeclaration/fixtures/_error_/implements-non-identifier/fixture.ts",
+    "declaration/ExportAllDeclaration/fixtures/_error_/non-string-source/fixture.ts",
+    "declaration/ExportAllDeclaration/fixtures/_error_/type-kind/fixture.ts",
+    "declaration/ExportNamedDeclaration/fixtures/_error_/anonymous-class/fixture.ts",
+  },
+  "TSESTree errored but Babel didn't": Set {
+    "declaration/ExportAllDeclaration/fixtures/_error_/named-non-identifier/fixture.ts",
+    "declaration/ExportNamedDeclaration/fixtures/_error_/aliased-literal/fixture.ts",
+  },
+}
+`;
```

Comment:
snapshot which provides a centralised list of fixtures with error differences (not error messages, just existence of errors)